test/e2etest.py

```diff
@@ -1237,6 +1237,12 @@ def test_post_incident(sample_user, sample_team, sample_application_name, sample
     assert re.status_code == 200
     assert re.json() == {'owner': sample_user, 'incident_id': incident_id, 'active': False}
 
+    # Invalid claim owner
+    re = requests.post(base_url + 'incidents/%d' % (incident_id, ), json={
+        'owner': 'nonexistent-user-foobar',
+    }, headers={'Authorization': 'hmac %s:abc' % sample_application_name})
+    assert re.status_code == 400
```

Comment:
Can you check that we're getting back the error message `Invalid claim: no matching owner` in addition to the 400? This way we can validate against other types of 4xxs.